jekyll/_cci2/runner-api.adoc

```diff
@@ -0,0 +1,190 @@
+= CircleCI Runner API
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+//:circle-success: image:circle-success.png[]
+//:circle-warning: image:circle-warning.png[]
+//:circle-failure: image:circle-failure.png[]
```

Comment:
These images were for a specific server page so these lines can safely be removed :-)

Reply:
Fixed